net/netatalk/Portfile

```diff
@@ -1,81 +1,68 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           openssl 1.0
+PortGroup           github 1.0
```

Comment:
`netatalk4` needs `legacysupport` for `openat` (at least).

Reply:
Again I'm not clear on why this code is shown. How would legacysupport alter the build configurations is my concerns. I don't want this software to pickup old libraries on modern systems.